platform/src/main/java/org/stellar/anchor/platform/controller/Sep31Controller.java

```diff
@@ -1,10 +1,14 @@
 package org.stellar.anchor.platform.controller;
 
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import static org.stellar.anchor.platform.controller.Sep10Helper.getSep10Token;
+
+import javax.servlet.http.HttpServletRequest;
+import lombok.SneakyThrows;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
 import org.stellar.anchor.dto.sep31.Sep31InfoResponse;
+import org.stellar.anchor.dto.sep38.Sep38QuoteResponse;
+import org.stellar.anchor.sep10.JwtToken;
```

Comment:
🤔  Why did the imports change so much? Are you sure all these imports are being used?